### PR TITLE
Adding Latvian translations

### PR DESCRIPTION
**Pull request by marcomow**
# Explain your changes

Adding Latvian translations

# Checklist

- [x] I have read the [“Pull requests” section](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md#pull-requests) in the [contributing guidelines](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I agree to the terms within the [code of conduct](https://github.com/kinde-oss/.github/blob/main/.github/CODE_OF_CONDUCT.md).

🛟 _If you need help, consider asking for advice over in the [Kinde community](https://thekindecommunity.slack.com)._

## Summary by CodeRabbit

- **Documentation**
	- Added an emoji symbol "✅" next to the Latvian language entry to indicate its completion.

### CODE REVIEW

**Comment by Patsy101** [comment]
Hi @marcomow  can you add  a check mark on read me file ✅ this is just for better reading of readme file
image

Cheers,
@Patsy101

**Comment by marcomow** [comment]
Sure thing, done @Patsy101

File: lv/auth.json [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+      "credit_logo_link": "ApmeklÄ“t Kinde",
+      "skip_link_main": "PÄriet uz galveno saturu"
+    },
+    "mfa_sms_page": {
+      "page_title": "Ievadiet tÄlruÅ†a numuru | ${business_name}",
+      "heading": "AutentifikÄcija caur SMS",
+      "description": "Pievienojiet tÄlruÅ†a numuru, lai saÅ†emtu kodus ar sms",
+      "continue_button": "TurpinÄt",
+      "phone_input_label": "TÄlruÅ†a numurs",
+      "phone_input_help_text": "AtstÄjiet ÄrÄ sÄkuma 0",
```
[the remark this answers is not in the document]
Comment (marcomow) [reply lv/auth.json:20]: yes something definetly broke with the encoding, I'm going to fix it

[the remark this answers is not in the document]
Comment (marcomow) [reply lv/auth.json]: @evgenyk fixed!

**Review by evgenyk — APPROVED** [review]
LGTM, @DaveOrDead could you please have a look?

**Pull request merged**